docs/details/commands/pmm-admin.md

```diff
@@ -344,18 +362,43 @@ For example, if you want all collectors except `topmetrics`, specify:
 
 ###### Limit dbStats, collStats and indexStats
 
- All the collectors are enabled by default. However, collStats and indexStats are enabled if there are less than 200 collections across all the databases, excluding the admin, config, and local system databases.
+By default PMM decides what limit to set for the number of collections to monitor for the collStats and indexStats collectors. 
```

Comment:
We could rephrase as below:

BY default, PMM decides the limit for the number of collections to monitor for the collStats and indexStats collectors.